Automattic/wp-calypso
# Add Dot Pager component

#### Changes proposed in this Pull Request

Adds a `DotPager` component to display a browsable container that renders only one child at time.

![Apr-15-2020 12-02-46](https://user-images.githubusercontent.com/1233880/79325248-28913d00-7f11-11ea-86ab-21df137a00ad.gif)


#### Testing instructions

- Go to `/devdocs/design/dot-pager`.
- Make sure the dot pager looks like the mockup in #41037

Part of #41037
